ext/hook/uhook.c

```diff
@@ -238,6 +238,17 @@ static void dd_zend_interrupt_function(zend_execute_data *ex)
 }
 #endif
 
+void dd_uhook_log_invocation(void (*log)(const char *, ...), zend_execute_data *execute_data, const char *type, zend_object *closure) {
+    const zend_function *func = zend_get_closure_method_def(closure);
+    log("Running a %s hook function from %s:%d on %s %s%s%s",
```

Comment:
Why are we using `log` here instead of `LOG`? Does it matter?

Reply:
the `log` variable has the log level included and is spawned into your scope when using LOGEV.